invenio_rdm_records/services/schemas/parent/access.py

```diff
@@ -80,9 +80,8 @@ class ParentAccessSchema(Schema, FieldPermissionsMixin):
 
     field_dump_permissions = {
         # omit fields from dumps except for users with 'manage' permissions
-        # allow only 'settings'
+        # allow only 'settings' and 'owned_by'
         "grants": "manage",
-        "owned_by": "manage",
```

Comment:
This is to expose the `owned_by.user.id` in the Zenodo Legacy REST API.

Besides that, another argument for exposing it could be that this doesn't really "leak" any private information. Someone can with some creative detective work figure out via a search query a user's records to find their user ID.

Reply:
why does owners' user id need to be exposed in the api ?

Reply:
> why does owners' user id need to be exposed in the API ?

Main reason is I need it at the Zenodo legacy REST API serializations, and if it's not dumped here, it's not available later on at the response serializer.

Besides that, I don't see an issue with exposing it here. User IDs can be discovered via search queries (one can do a binary search to find the user ID of a record a la `id:<record_id> AND parent.access.owned_by.user:(1 to 1000)`), so we're just obfuscating here, and not hiding.